main.go

```diff
@@ -145,3 +126,26 @@ func run(params commandParams) (*ldtest.Results, error) {
 
 	return &results, nil
 }
+
+func loadSuppressions(params *commandParams) error {
+	file, err := os.Open(params.suppressFailures)
+	if err != nil {
+		return fmt.Errorf("cannot open provided suppression file: %v", err)
+	}
+	defer func() {
+		if err := file.Close(); err != nil {
+			fmt.Fprintf(os.Stderr, "Failed to close suppressions file: %v\n", err)
```

Comment:
Despite what the articles say about the reasons to always check the return value of Close(), I honestly feel this is overkill. I mean... if there does happen to be a freak occurrence of the last write failing, there is absolutely nothing we can do about it and the error message isn't going to help us at all, we still have an incomplete file. But printing this doesn't hurt anything, I guess.